src/utils/uid.js

```diff
@@ -8,7 +8,7 @@ import generate from 'uid'
  */
 
 function uid() {
-  return generate(4)
+  return String(N++)
```

Comment:
hmm...
Well the persisted state does not store the keys. So you could have two instances of the Slate library loaded. There's only one place where we re-keys the nodes as needed, that's in the `Node.insertNode` method. So maybe you are bypassing that method somehow too ?